tailwind.config.js

```diff
@@ -7,18 +7,20 @@ module.exports = {
   darkMode: false, // or 'media' or 'class'
   theme: {
     fontFamily: {
-      sans: ['"IBM Plex Sans"', 'sans-serif'],
+      sans: ['"IBM Plex Sans"', 'sans-serif']
     },
     extend: {
       colors: {
-        primary: '#ff00af',
+        primary: '#ff00af'
       }
     }
   },
   variants: {
-    extend: {}
+    extend: {
+      borderRadius: ['first', 'last']
```

Comment:
what is this for, if not using, revert it

Reply:
This is used in my `renderBlockDetails`
```
      <div className='px-6 py-4 border first:rounded-t-md last:rounded-b-md flex justify-between' key={d.label}>
        <span className='w-1/3 flex-shrink-0'>
          {d.label}
        </span>
```
The plugin needs to be enabled according to tailwindcss [docs](https://tailwindcss.com/docs/hover-focus-and-other-states#first-child)

Reply:
don't really need this, border-radius parent box, and divider between the sub sections

Reply:
Revert this file, best not to extend features that we don't need.